.yamato/urp_lighting-win-dx11.yml

```diff
@@ -140,9 +140,10 @@ URP_Lighting_Win_DX11_playmode_cache_mono_Linear_trunk:
 URP_Lighting_Win_DX11_editmode_mono_Linear_trunk:
     name: URP_Lighting on Win_DX11_editmode_mono_Linear on version trunk
     agent:
-        type: Unity::VM
+        type: Unity::VM::GPU
         image: sdet/gamecode_win10:stable
-        flavor: b1.large
+        flavor: b1.xlarge
```

Comment:
@sophiaaar FYI ^

Reply:
Changed it back to b1.large as requested by @theo-pnv

Reply:
For playmode and standalone jobs the GPU is needed (with `b1.large` flavor) but for editmode we don't need the GPU though. So you can keep `Unity::VM` and `b1.xlarge` for those.

Reply:
The new reflection probe tests are edit mode, so they need a GPU right?

Reply:
Yes, thats true. I also want to add GPU LM tests to that suite.